util/Radiance_Monitor/image_gen/ush/mk_angle_plots.sh

```diff
@@ -260,7 +307,10 @@ for sat in ${big_satlist}; do
          (( ii=ii+1 ))
       done
 
-      if [[ ! $MY_MACHINE = "jet" ]]; then
+      if [[ $MY_MACHINE = "hera" ]]; then
+         $SUB --account ${ACCOUNT} -n $ii  -o ${logfile} -D . -J ${jobname} --time=4:00:00 \
```

Comment:
Why is --mem=0 required?  Wouldn't all of the memory for the node be made available anyway?  Or is it different for MPMD tasks?

Reply:
Apparently there was an upgrade to the batch processing software on hera somewhere around March 1, and this process began running out of memory.  You're right all the memory for the node used to be available by default but after the upgrade it apparently isn't.  I didn't go too far into the weeds but learned that, counterintuitively, with --mem=0 maximum memory for the node is made available.